### DISCUSSION TITLE
Is GH Discussions any good?

### DISCUSSION THREAD

**Discussion by mvdan** [cuelang/cue#385] [General] [answered]
Input welcome.

**Comment by myitcv** [comment, marked as answer]
Looks good to me.

**Reply by mvdan** [reply]
Marked as an answer, which I think is good for visibility?

**Comment by mpvl** [comment]
Looks great, I'll leave this test for now and remove once we a few discussions going.

**Comment by mvdan** [comment]
Oh, neat, I can leave emojis on comments too.

**Comment by verdverm** [comment]
This seems like a great way to preserve the important conversations that get lost on Slack. If they added chatroom like channels here, would we start leaving Slack?

**Reply by mpvl** [reply]
Yes, I think that for many of the discussions this would be a great forum. Anything where it's worth preserving content.

We are considering leaving Slack for Discord for remaining discussions. Discord has a lower barrier to entry (has a great web client) and better communication means for an unpaid setup. Also MD rendering is superior. At the same time, it has no threading. This is great, as it will put more social pressure to move "discussions" to Discussions.

**Reply by verdverm** [reply]
I like Gitter more too, great integrations with GitHub

**Comment by proppy** [comment]
This also look like a nice way prevent `issues` from being filed with general usage questions.

**Reply by proppy** [reply]
I wonder if there are tools to migrate content between the two namespaces? ex:
- move an issue to a discussion
- file an issue from a discussion

**Reply by myitcv** [reply]
> This also look like a nice way prevent `issues` from being filed with general usage questions.

That's the idea!

> * move an issue to a discussion
> * file an issue from a discussion

For repository owners/maintainers etc this already exists in a limited form, and the GitHub team are looking to make it even better.

**Comment by cueckoo** [comment]
This discussion has been migrated to https://github.com/cue-lang/cue/issues/385.

For more details about CUE's migration to a new home, please see https://github.com/cue-lang/cue/issues/1078.
